src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1216,6 +1216,7 @@ int shim::cmaEnable(bool enable, uint64_t size)
         uint64_t allocated_size = 0;
         uint32_t page_num = size >> 30;
         drm_xocl_alloc_cma_info cma_info = {0};
+        uint64_t user_addr[page_num] = {0};
```

Comment:
This is not standard, please use std::vector

Reply:
I forgot this rule. std::vector is even better. Thanks for pointing it out. I will fix this.